bazel/repository_locations.bzl

```diff
@@ -304,30 +368,47 @@ REPOSITORY_LOCATIONS = dict(
         # Only allow peeking at fuzzer related files for now.
         strip_prefix = "compiler-rt-9.0.0.src/lib",
         urls = ["http://releases.llvm.org/9.0.0/compiler-rt-9.0.0.src.tar.xz"],
+        use_category = ["build"],
     ),
     fuzzit_linux = dict(
         sha256 = "9ca76ac1c22d9360936006efddf992977ebf8e4788ded8e5f9d511285c9ac774",
         urls = ["https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.76/fuzzit_Linux_x86_64.zip"],
+        use_category = ["build", "test"],
     ),
     upb = dict(
         sha256 = "e9f281c56ab1eb1f97a80ca8a83bb7ef73d230eabb8591f83876f4e7b85d9b47",
         strip_prefix = "upb-8a3ae1ef3e3e3f26b45dec735c5776737fc7247f",
         # 2019-11-19
         urls = ["https://github.com/protocolbuffers/upb/archive/8a3ae1ef3e3e3f26b45dec735c5776737fc7247f.tar.gz"],
+        use_category = ["api"],
```

Comment:
This is for gRPC, we don't use it directly. I'd call it as a potential dataplane/controlplane. The rationale for dataplane is that we make sidecalls with this, so crashes could be problematic.

Reply:
done